src/pytest_postgresql/executor.py

```diff
@@ -136,19 +136,24 @@ def init_directory(self):
         # remove old one if exists first.
         self.clean_directory()
         init_directory = (
-            self.executable, 'initdb',
-            '-o "--auth=trust --username=%s"' % self.user,
-            '-D %s' % self.datadir,
+            self.executable, 'initdb', '-D {}'.format(self.datadir)
```

Comment:
prefer f strings if migrating to other formatting type.

Reply:
Sure, changed.